lrpar/src/lib/ctbuilder.rs

```diff
@@ -256,6 +235,13 @@ where
         self
     }
 
+    pub fn conflicts(&self) -> Option<(&YaccGrammar<StorageT>, &StateGraph<StorageT>, &StateTable<StorageT>, &Conflicts<StorageT>)> {
+        if let Some((ref grm, ref sgraph, ref stable)) = self.cdata {
+            return Some((&grm, &sgraph, &stable, &stable.conflicts().unwrap()));
```

Comment:
I'm not sure you need the `&` on `grm` and `sgraph`.

Reply:
And you are right again! :-)